lib/engine.js

```diff
@@ -14,30 +14,24 @@
 
 //const debug = require('debug')('pipeline:engine');
 const debug = console.log;
-//const http = require('./sdk/storage/http');
 const PlanFinder = require('./plan-finder');
 const {Plan} = require('./plan');
-const {createBaseDirectory, createDirectories, cleanupDirectories} = require('./sdk/prepare');
-const Asset = require('./sdk/asset');
-const Rendition = require('./sdk/rendition');
+const Asset = require('./asset');
+const Rendition = require('./rendition');
+const utils = require('./utils');
+const Timer = utils.Timer;
```

Comment:
I would prefer to have these in one import statement if possible

Reply:
Was doing that at first but thought that since it is at the same level... so did it this way instead.